### PR TITLE
Update version to v0.2.3

### PR DESCRIPTION
**Pull request by mdubus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,4 +106,4 @@ If you want to contribute to this project, please make sure to read [the contrib
 
 ## Compatibility with Meilisearch
 
-The current version of the mini-dashboard only guarantees the compatibility with the [version v0.28.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.28.0).
+The current version of the mini-dashboard only guarantees the compatibility with the [version v0.29.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.29.0).
```
Comment (bidoubiwa) [inline README.md:109]:
```suggestion
The current version of the mini-dashboard only guarantees the compatibility with the [version v0.29.1 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.29.1).
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
🔥

**Comment by mdubus** [comment]
bors merge

**Comment by mdubus** [comment]
bors try

**Comment by mdubus** [comment]
bors merge

**Pull request merged**